### PR TITLE
Adds Axolotls to List of Cytology Creatures

### PR DESCRIPTION
**Pull request by ophaq**
## About The Pull Request
Adds axolotls to list of cytology creatures. Edits the name of the cells for frogs by changing to "anura amphibian cells" and for axolotls, "caudata amphibian cells." These mean both "frog" and "salamander" amphibian cells respectively. Also makes axolotls swabbable, adds their cell lines to fern's list, and they are now distinguishable from the frog's reagent lists.
## Why It's Good For The Game
Adding more thing to cytology is always good. Frogs could be grown but not axolotls despite them coming from the same place and are amphibians. Adds more ways to get axolotls and another reason why you should do cytology.

Reinforces the PR ([https://github.com/tgstation/tgstation/pull/72922](url)) I added for amphibian cytology.
## Changelog
:cl:
add: Axolotl lovers rejoice! I added axolotls to cytology and as such are now swabbable.
add: Frog cells are now named "anura amphibian cells." This literally means "frog amphibian cells."
add: Axolotl are "caudata amphibian cells." This literally means "salamander amphibian cells."
/:cl:

### CODE REVIEW

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 1]
```diff
@@ -583,6 +583,27 @@
...
+	supplementary_reagents = list(
+		/datum/reagent/ants = 3,
+		/datum/reagent/consumable/eggwhite= 2,
+		/datum/reagent/consumable/nutriment/vitamin = 1,)
+
+	suppressive_reagents = list(
+		/datum/reagent/toxin/carpotoxin = -3,
+		/datum/reagent/toxin/bungotoxin = -3,
+		/datum/reagent/toxin/spore = -3,
+		/datum/reagent/toxin/plantbgone = -2, //GAY FROGS
```
Comment (kawoppi) [inline code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:599]:
```suggestion
		/datum/reagent/toxin/plantbgone = -2, //GAY AXOLOTLS
```

Comment (ophaq) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm]: That's a good idea. Wasn't sure on // so I left it alone. Minor change that will surely not destroy an entire station.

**Comment by kawoppi** [comment]
Doesn't this mean that you would need an axolotl to swab before you can start making axolotls with cytology? It might be handy to add the axolotl cell line to `CELL_LINE_TABLE_ALGAE` so that you can get them from swabbing neglected ferns, just like the frog cell line. Although I'm not sure if that would conflict with the frog cell line, since both cell lines have the same required/supplementary/suppressive reagents

**Comment by ophaq** [comment]
> Doesn't this mean that you would need an axolotl to swab before you can start making axolotls with cytology? It might be handy to add the axolotl cell line to `CELL_LINE_TABLE_ALGAE` so that you can get them from swabbing neglected ferns, just like the frog cell line. Although I'm not sure if that would conflict with the frog cell line when growing, since both cell lines have the same required/supplementary/suppressive reagents

You could, but with the PR I referenced and both frogs and axolotls being spawnable from moisture traps during the round, that is usually the main method most players would get them I believe.

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 2]
```diff
@@ -583,6 +583,27 @@
...
+		/datum/reagent/consumable/nutriment/vitamin = 1,)
+
+	suppressive_reagents = list(
+		/datum/reagent/toxin/carpotoxin = -3,
+		/datum/reagent/toxin/bungotoxin = -3,
+		/datum/reagent/toxin/spore = -3,
+		/datum/reagent/toxin/plantbgone = -2, //GAY AXOLOTLS
+		/datum/reagent/drying_agent = -2,
+		/datum/reagent/consumable/mold = -2,
+		/datum/reagent/toxin = -1)
```
[the remark this answers is not in the document]
Comment (ophaq) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:602]: > I'm not too familiar with cytology but straight up copying the reagents list from frogs isn't good I don't think. If you get a sample with both cells in it the recipes will conflict?
> 
> I think you need to change some of the reagents to fit better.

I tested on a test server the supplementary reagent list. The vat spawned the axolotl with the cell petri dish with the required reagents as well as when tested with the frog. Both are amphibians, so it made sense to use the same reagents.

[the remark this answers is not in the document]
Comment (Comxy) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:602]: I feel like there should at least be one different chemical in supplementary_reagents and suppressive_reagents  to differentiate between the two cell lines

**Comment by ophaq** [comment]
New update because people wanted it. I tested it too and I got it from the fern, plus they work normally like how you would do cytology.

Adds line to "CELL_LINE_TABLE_ALGAE = list" for axolotl in xenobiology.dm so they can be swabbable from ferns.
Changes lines in both supplemental and suppressive reagent lists for axolotl in common.dm to distinguish between frog and axolotl with the following...
Supplemental added/changed:
/datum/reagent/liquidgibs = 2,
/datum/reagent/consumable/salt = 1,
/datum/reagent/consumable/eggwhite= 1,

Suppressive added/changed:
Carpotoxin was replaced with "/datum/reagent/ammonia = -3,"
Change from -3 to -4 value into "/datum/reagent/drying_agent = -4,"

**Pull request merged**